stanford-earth/matson
# Earth 698 earth display

# NOT READY

# Summary
- Change the order of fields in the Earth Events Display and do re-styling as needed.

# Needed By (Date)
- End of sprint

# Urgency
- Medium

# Steps to Test

1. Go to an event node
2. Take alook to see that When, Where, etc. are up under the title
3. Less important info (audience, etc.) is below the description
4. Social media links are moved under the image instead of under the title/subtitle info

# Affected Projects or Products
- I added some general styling to social actions class since it is used multiple places. I think more consolidation is possible.

# Associated Issues and/or People
- EARTH 698

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
